scripts/snowflake_generate_grant_all/go.mod

```diff
@@ -0,0 +1,45 @@
+module github.com/chanzuckerberg/cztack/snowflake_generate_grant_all
```

Comment:
Is there a reason not to "just" use the top-level modules file?

Reply:
what do you mean by that?

Reply:
oh, I get it now. I did it this way to have a separate `main.go` for this script only but I could definitely be missing some set of configs that make it possible to share the mod files.

Reply:
I think you can have more than one main.go with the same go.mod right?